new-pipeline/src/dist.js

```diff
@@ -9,11 +9,30 @@ var gPreviousFilterAllSelected = {};
 var gAxesList;
 var gAxesSelectors;
 var gNoneKey = "__none__";
+var changePending = false;
 
 indicate("Initializing Telemetry...");
-
+function stopProgress() {
+  clearTimeout(gFilterChangeTimeout);
+}
+function startProgress() {
+  if(changePending) {
+    gFilterChangeTimeout = setTimeout(function () {
+      $('#measure')
+        .trigger('change');
+      changePending = false;
+      }, 3000);
+  }
+}
 $(function () {
   Telemetry.init(function () {
+    let btnGroup = $('.btn-group');
+    btnGroup.on('hide.bs.dropdown', function () {
+      startProgress();
+    });
+    btnGroup.on('click', function () {
```

Comment:
Yes, there is a `show.bs.dropdown`, but I can't use that here. 
_____
https://github.com/mozilla/telemetry-dashboard/blob/66ed4e9dcd263f563bd8361fba8a507e2c19367c/lib/bootstrap-multiselect.js#L353
Here on `show.bs.dropdown`(triggered by metricsgraphics.js) the function `onDropdownShow` is triggered which is used here
https://github.com/mozilla/telemetry-dashboard/blob/f66351757f13829807efd3c661ae28a0bcf5899a/new-pipeline/src/dashboards.js#L18
So if I add an event listener for the same then this would overwrite the previous event listener
______
https://github.com/mozilla/telemetry-dashboard/blob/66ed4e9dcd263f563bd8361fba8a507e2c19367c/lib/bootstrap-multiselect.js#L354
But here no overriding takes place thus I have used it